lib/cocoapods-core/specification/dsl.rb

```diff
@@ -126,6 +126,25 @@ module DSL
       root_attribute :version,
                      :required => true
 
+      #------------------#
+
+      # @!method swift_version=(version)
+      #
+      #   The version of Swift that the specification supports.
+      #
+      #   @example
+      #
+      #     spec.swift_version = '>= 3.2'
+      #     spec.swift_versions = ['>= 3.2', '< 4.0']
+      #
+      #   @param  [String] swift_version
+      #
+      root_attribute :swift_versions,
+                     :container => Array,
+                     :singularize => true,
+                     :multi_platform => false,
+                     :root_only => true
```

Comment:
nit: I think `root_attribute` by default has this property set to true.

Reply:
yup